src/Hooks/useInputState.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react";
+
+function UseInputState(initialValue) {
+    const [value, setValue] = useState(initialValue);
+    const change = (e) => {
+        setValue(e.target.value);
```

Comment:
This is a syntax sugar tip. Arrow functions that have a single line don't need curly braces. This could even be written as
```
const change = (e) => setValue(e.target.value)
```